test/functional/normal/jump_spec.lua

```diff
@@ -0,0 +1,50 @@
+local helpers = require('test.functional.helpers')(after_each)
+
+local clear = helpers.clear
+local command = helpers.command
+local eq = helpers.eq
+local funcs = helpers.funcs
+local feed = helpers.feed
+local write_file = helpers.write_file
+
+describe("Jumplist", function()
+  local fname1  = "Xtest-functional-normal-jump"
+  local fname2 = fname1 .. "2"
+
+  setup(function()
+    write_file(fname1,  "first file contents");
+    write_file(fname2, "second file contents");
```

Comment:
do this at the start of the `it()` block unless these files are useful for other `it()` cases.